controllers/machineset_controller_test.go

```diff
@@ -690,108 +690,6 @@ func TestAdoptOrphan(t *testing.T) {
 	}
 }
 
-func TestHasMatchingLabels(t *testing.T) {
```

Comment:
Do all these tests that we're removing have a counterpart?

Reply:
yeh, see https://github.com/kubernetes-sigs/cluster-api/pull/4652/files#diff-0852b95be749651472935b5917c87dc0447b365853342b008761afebd91713dcL27